src/pages/FuFit.jsx

```diff
@@ -0,0 +1,184 @@
+{/*  */}
+import Navbar from "../components/Navbar";
+import Footer from "../components/Footer"
+import fufit_navbar from "../assets/fufit_navbar.png"
+import fufit_basketball from "../images/fufit_basketball.png"
+import fufit_fever from "../images/fufit_fever.png"
+import fufit from "../assets/fufit.png"
+import fufit_destress from "../images/fufit_destress.png"
+import fufit_nutrition from "../images/fufit_nutrition.png"
+import fufit_bg1 from "../assets/fufit_bg1.png"
+import fufit_bg2 from "../assets/fufit_bg2.png"
+import fufit_bg3 from "../assets/fufit_bg3.png"
+import fufit_bg4 from "../assets/fufit_bg4.png"
+import fufit_shadow1 from "../assets/fufit_shadow1.png"
+import fufit_shadow2 from "../assets/fufit_shadow2.png"
+import fufit_dots1 from "../assets/fufit_dots1.png"
+import fufit_dot1 from "../assets/fufit_dot1.png"
+
+
+
+
+export function FuFit() {    
+    return (
+        <div className="flex flex-col">
+            <Navbar title="FUSION FIT" subtitle="PROGRAMS" bg={fufit_navbar} />
+
+            {/* FuFuit Overview */}
+            <section className="w-full h-[540px] bg-white flex justify-center items-center">
+                <div className="w-[58%] h-1/2 text-center">
+                    <div className="font-semibold text-5xl">
+                        <span className="text-black">Embrace your</span> <span className="text-[#FFA523]">Inner Athelete</span>
+                    </div>
+                    <hr className="w-48 h-1 mx-auto bg-[#FAC80A] border-0 rounded my-10"/>
+                    <div className="h-1/2 flex justify-center items-center">
+                        <p className="text-black text-center">
+                        FUSION's athletic programs aim to build a strong community, enhance physical well-being, and support mental 
+                        health through a range of recreational and organized activities. By providing opportunities for exercise, we 
+                        encourage our members to stay active and healthy. Our organized sports activities foster teamwork and camaraderie, 
+                        while our workshops offer a supportive space for relaxation and emotional well-being. Together, these programs 
+                        ensure that our members have the resources they need to thrive both physically and mentally.
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            {/* Get moving */}
+            <section className="w-full h-[540px] bg-white flex flex-row relative">
```

Comment:
I was playing around with your code, lowkey the h-[540px] exposes a lot of what's happening under the hood for your code, remove this and see what happens to your elements. From there make necessary adjustments